### PR TITLE
Portfolio: Application deploy

### PR DESCRIPTION
**Pull request by Alusp**
## In this PR:

- I  added the online version of the project to git hub pages as requested and below is the following link as well:

[alusine-jalloh-portfolio](https://alusp.github.io/My-Portfolio/)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -30,8 +30,10 @@ Navigate through the web page to find the information you need. Use the links in
 
 
 ### Deployment
-  Project reside in this Repo. You can clone the repo or download the files manually to your local machine.
+  Project reside in this Repo and is hosted on netlify. You can clone the repo or download the files manually to your local machine.
+  below is the online link for this website:
```
Comment (Ginohmk) [inline README.md:34]: - PLease as stated in the [requirement](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/5_deploy.md), you are to make use of GitHub pages and not netlify for this project, please make use of GitHub pages to deploy your webpage.

**Review by Ginohmk — CHANGES_REQUESTED** [review]
Hi @Alusp, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
you-are-near

## Highlights
- No linter error :+1: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- x
- y
- z

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (Alusp) [reply README.md]: ### Fixes applied according to the requirement

**Review by Bondok6 — APPROVED** [review]
# STATUS: APPROVED :green_circle: :clap: 

## Hi @Alusp  🙋‍♂️, 
Your project is complete!  There is nothing else to say other than... Great job 👏 
Congratulations! 🎉 

TO HIGHLIGHT:

- Good Summary Description ✅
- The website is deployed using GitHub pages correctly ✅
-  Professional README.md file 💯

Now you can merge this branch and move on :1st_place_medal: 
HAPPY CODING AND GOOD LUCK WITH THE COMMING ONES :crossed_fingers: :four_leaf_clover: 



## Optional Suggestions
- [OPTIONAL] I suggest adding `MIT.md` file to your project 👌 [Microverse Template](https://github.com/microverseinc/readme-template)
> The MIT license gives users express permission to reuse code for any purpose, sometimes even if the code is part of proprietary software. As long as users include the original copy of the MIT license in their distribution, they can make any changes or modifications to the code to suit their own needs.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**